src/v1/driver.js

```diff
@@ -224,11 +236,19 @@ class Driver {
    * @return undefined
    */
   close() {
-    for (let sessionId in this._openSessions) {
-      if (this._openSessions.hasOwnProperty(sessionId)) {
-        this._openSessions[sessionId].close();
-      }
+    this._log.info(`Driver ${this._id} closing`);
+
+    try {
+      // purge all idle connections in the connection pool
       this._pool.purgeAll();
+    } finally {
```

Comment:
Why are these changes part of this PR?

Reply:
I noticed a double-close of connections because of logging and decided to fix it in-place :)
Will move it to a different PR.

Reply:
Extracted two separate commits - one for `RoutingTable#toString()` change and one about this `Driver#close()`

Reply:
👍